app/models/maintenance_tasks/run.rb

```diff
@@ -34,10 +34,12 @@ class Run < ApplicationRecord
       Task.available_tasks.map(&:to_s)
     } }
     validate :csv_attachment_presence, on: :create
+    validate :valid_task_arguments?, on: :create
```

Comment:
```suggestion
    validate :validate_task_arguments, on: :create
```

Validations are actions, not queries

Reply:
Opting for `task_arguments_are_valid` to make it less redundant